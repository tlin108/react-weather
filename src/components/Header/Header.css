/*
 * Header Styles
 */
.site-header {
  padding: 50px 0; }
  .site-header .branding {
    float: left; }
    .site-header .branding .logo, .site-header .branding .logo-type {
      display: inline-block;
      vertical-align: middle; }
    .site-header .branding .logo {
      margin-right: 10px; }
    .site-header .branding .site-title {
      margin-bottom: 5px;
      font-size: 16px;
      font-size: 1.1428571429em;
      color: white; }
    .site-header .branding .site-description {
      font-size: 10px;
      font-size: 0.7142857143em;
      display: block;
      color: #bfc1c8; }