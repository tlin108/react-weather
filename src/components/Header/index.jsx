import React from 'react';
import './Header.css';

export default function Header() {
  return (
    <div className="site-header">
      <div className="container">
        <a href="index.html" className="branding">
          <img src="images/logo.png" alt="" className="logo" />
          <div className="logo-type">
            <h1 className="site-title">SkyCast</h1>
            <small className="site-description">Weather at your finger tip</small>
          </div>
        </a>
      </div>
    </div>
  );
}
