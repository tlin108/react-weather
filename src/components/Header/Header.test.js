import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

it('renders without crashing', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
