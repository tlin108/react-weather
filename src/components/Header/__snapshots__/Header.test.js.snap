exports[`test renders without crashing 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="site-header">
    <div
        className="container">
        <a
            className="branding"
            href="index.html">
            <img
                alt=""
                className="logo"
                src="images/logo.png" />
            <div
                className="logo-type">
                <h1
                    className="site-title">
                    SkyCast
                </h1>
                <small
                    className="site-description">
                    Weather at your finger tip
                </small>
            </div>
        </a>
    </div>
</div>,
  "nodes": Array [
    <div
      className="site-header">
      <div
            className="container">
            <a
                  className="branding"
                  href="index.html">
                  <img
                        alt=""
                        className="logo"
                        src="images/logo.png" />
                  <div
                        className="logo-type">
                        <h1
                              className="site-title">
                              SkyCast
                        </h1>
                        <small
                              className="site-description">
                              Weather at your finger tip
                        </small>
                  </div>
            </a>
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Header />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="site-header">
          <div
                    className="container">
                    <a
                              className="branding"
                              href="index.html">
                              <img
                                        alt=""
                                        className="logo"
                                        src="images/logo.png" />
                              <div
                                        className="logo-type">
                                        <h1
                                                  className="site-title">
                                                  SkyCast
                                        </h1>
                                        <small
                                                  className="site-description">
                                                  Weather at your finger tip
                                        </small>
                              </div>
                    </a>
          </div>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="site-header">
          <div
                    className="container">
                    <a
                              className="branding"
                              href="index.html">
                              <img
                                        alt=""
                                        className="logo"
                                        src="images/logo.png" />
                              <div
                                        className="logo-type">
                                        <h1
                                                  className="site-title">
                                                  SkyCast
                                        </h1>
                                        <small
                                                  className="site-description">
                                                  Weather at your finger tip
                                        </small>
                              </div>
                    </a>
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Header />,
}
`;
